src/args/arg.rs

```diff
@@ -3711,9 +3712,20 @@ impl<'a, 'b> Arg<'a, 'b> {
         self
     }
 
+    /// Indicates that all parameters passed after this should not be parsed
+    /// individually, but rather passed in their entirety.
+    ///
+    /// **NOTE:** Implicitly sets [`Arg::multiple(true)`], [`Arg::allow_hyphen_values(true)`], and
+    /// [`Arg::last(true)`] when set to `true`
```

Comment:
Links may need to be added here. I know Rust is working on doing this automatically, but until I'm sure that works using manual links will have to do (i.e.):

```
/// [`Arg::last(true)`]: ./struct.Arg.html#method.last
```

At the bottom of the doc comment. You'll have to do this for each of the links above it (`[foo]`)